extensions/amp-date-display/1.0/date-display.type.js

```diff
@@ -22,13 +22,10 @@ var DateDisplayDef = {};
 /**
  * @typedef {{
  *   children: (?PreactDef.Renderable|undefined),
- *   datetime: (string|undefined),
+ *   datetime: (?Date|number|undefined),
```

Comment:
Should we be supporting `string`, too?

Reply:
What do you think? We definitely could. The main goal was to reduce number of props. I think that buys us more room with what we do with types.

Reply:
I added support for a string.